config.sh

```diff
@@ -12,3 +12,16 @@ function run_tests {
     python --version
     pytest -l --pyargs sklearn
 }
+
+function enable_openmp {
+    # Install OpenMP
+    brew install libomp
+    export CC="clang -Xpreprocessor -fopenmp"
+    export CFLAGS="$CFLAGS -I/usr/local/opt/libomp/include"
```

Comment:
Maybe we should also deal with the case where we want openmp in C++ source:

```sh
export CXX="clang++ -Xpreprocessor -fopenmp"
export CXXFLAGS="$CXXFLAGS -I/usr/local/opt/libomp/include"
```

I have not tested but it seems to be valid according to: https://stackoverflow.com/a/39983933/163740

Reply:
distutils does not check the `CXX` variable, so it would have no effect.
Besides `clang++` is just a symlink to clang :)

Reply:
> distutils does not check the CXX variable, so it would have no effect.

Apparently it is checked now. Maybe it's fixed in `numpy.distutils`, or maybe my info was just too old :/

> Besides clang++ is just a symlink to clang :)

This still holds.